File: C Programming/C programming Test 3/Ques1.c
//Ques1. WAP even and odd numbers in a given range.

void main()
{
	int start=10, end=25;
	
	printf("Odd num is:");
	for(int i=10; i<25; i++)
	{
		if(i%2!=0)
			printf("%d ",i);
	}
	printf("\n");
	
	int start1=1, end1=15;
	
	printf("Even num is:");
	for(int i=1; i<15; i++)
	{
		if(i%2==0)
			printf("%d ",i);
	}
}

File: C To Java/assignment1/Ques1.java
package assignment1;

public class Ques1 {
	
	public static void main(String[] args)
	{
		int num=7;
		int flag=0;
		
		for(int i=2; i<num; i++)
		{
			if(num%i==0)
			{
				flag=1;
				break;
			}
		}
		if(flag==0)
		{
			System.out.printf("Num is prime");
		}
		else
		{
			System.out.printf("Num is not prime");
		}
	}

}
